components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -24,6 +24,7 @@ export class CreateObjectModalComponent implements OnInit, OnDestroy, OnChanges
   @Input() createProjectModal = false;
   @Input() assignableProjects: Project[] = [];
   @Input() createForm: FormGroup; // NB: The form must contain 'name' and 'id' fields
+  @Input() resetCheckboxEvent: EventEmitter<any>;
```

Comment:
Should always avoid using "any" if there is an alternative, and in this case, this works as well:
```suggestion
  @Input() resetCheckboxEvent: EventEmitter<null>;
```